runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -531,4 +537,21 @@ private void resetTimers()
 			lastInteract = null;
 		}
 	}
+
+	@Subscribe
+	private void onVarbitChanged(VarbitChanged ev)
+	{
+		oxygenThreshold = config.oxygenAlertAmount();
+		if (oxygenThreshold >= 100 || oxygenThreshold < 0)
+		{
+			oxygenThreshold = 20;
+		}
+		if (client.getVar(Varbits.OXYGEN_LEVEL) <= oxygenThreshold * 10 && client.getVar(Varbits.OXYGEN_LEVEL) > 0)
+		{
+			if (oxygenThreshold == client.getVar(Varbits.OXYGEN_LEVEL) * 0.1)
+			{
+				notifier.notify("Your oxygen is dangerously low! Currently: " + client.getVar(Varbits.OXYGEN_LEVEL) * 0.1 + "%");
```

Comment:
Maybe use same format as the HP and Prayer notifier use

Reply:
I modified it to be close to the hp one.